### PR TITLE
update clean-webpack-plugin and webpack-bundle-analyzer dependencies.…

### PR DESCRIPTION
**Pull request by kentmclean**
… Have clean-webpack-plugin pipe to console when isDevServer == true

### CODE REVIEW

File: src/canopy-webpack-config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const path = require('path')
-const CleanWebpackPlugin = require('clean-webpack-plugin')
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
```
Comment (kentmclean) [inline src/canopy-webpack-config.js:2]: with 3.0.0 this changed from a default export to a named export

File: src/canopy-webpack-config.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ module.exports = function(name, overridesConfig) {
         ],
       },
       plugins: [
-        new CleanWebpackPlugin(),
+        new CleanWebpackPlugin({ verbose: isDevServer, }),
```
Comment (kentmclean) [inline src/canopy-webpack-config.js:65]: I want to see the delete in the console output when dev

File: src/canopy-webpack-config.js [hunk 3]
```diff
@@ -81,20 +81,20 @@ module.exports = function(name, overridesConfig) {
...
         /^rx$/,
         /^rxjs\/?.*$/,
-        /^sofe$/,
-        /^single-spa$/,
         /^single-spa-canopy$/,
-        /^prop-types$/,
-        /^canopy-sofe-extensions$/,
-        /^online-listener$/,
+        /^single-spa$/,
+        /^sofe$/,
```
Comment (kentmclean) [inline src/canopy-webpack-config.js:97]: tiny readability aid -- I reordered these alphabetically

**Review by TheMcMurder — APPROVED** [review]
:octopus:

**Pull request merged**